<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>271</width>
    <height>176</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="pb_create">
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pb_cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QCheckBox" name="cb_ramp">
      <property name="text">
       <string>Ramp (per sec)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QCheckBox" name="cb_scale">
      <property name="text">
       <string>Scale</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="PyScientificSpinBox" name="sb_ramp">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="PyScientificSpinBox" name="sb_scale">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="PyScientificSpinBox" name="sb_max">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="PyScientificSpinBox" name="sb_min">
      <property name="enabled">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" rowspan="2">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <widget class="QCheckBox" name="cb_limit">
        <property name="text">
         <string>Limit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lbl_min">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Min</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lbl_max">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Max</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PyScientificSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>pyHegel.gui.ScientificSpinBox</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
